.github/workflows/deltaengine.yaml

```
@@ -0,0 +1,160 @@
+
+  AssetBuild:
+    name: Asset-Build
+    needs:
+      - check_branch
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Set up JDK 21
+        uses: actions/setup-java@v2
```

actionlint

the runner of "actions/setup-java@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)